DOCS-8686 Workday 12.1 Edits (#1003)
mulesoft/docs-connectors: workday/12.1/modules/ROOT/pages/workday-reference.adoc

** SECONDS
** MINUTES
** HOURS
** DAYS |  Unit of time for Connection Idle Timeout |  SECONDS | {nbsp}
| Stream Response a| Boolean |  Whether or not to stream received responses, meaning processing continues as soon as all headers are parsed and the body is streamed as it arrives. When enabled, the response must eventually be read since, depending on the configured buffer size, it might not fit into memory and processing stops until space is available. |  false | {nbsp}

> | Response Buffer Size a| Number |  The space in bytes for the buffer where the HTTP response will be stored. |  -1 | {nbsp}

Reviewer: ```suggestion
| Response Buffer Size a| Number |  The space in bytes for the buffer where the HTTP response is stored. |  -1 | {nbsp}
```
